RNSentryAndroidTester/app/src/androidTest/java/io/sentry/rnsentryandroidtester/MapConverterTest.kt

```diff
@@ -0,0 +1,357 @@
+package io.sentry.rnsentryandroidtester
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.facebook.react.bridge.Arguments
+import com.facebook.soloader.SoLoader
+import io.sentry.react.MapConverter
+import org.junit.Assert.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import android.content.Context;
+import org.junit.Before
+import java.math.BigDecimal
+import java.math.BigInteger
+
+class Unknown
+
+
+@RunWith(AndroidJUnit4::class)
+class MapConverterTest {
+
+    @Before
+    fun setUp() {
+        val context: Context = InstrumentationRegistry.getInstrumentation().targetContext
+        SoLoader.init(context, false)
+    }
+
+    @Test
+    fun converts_unknown_value_to_null() {
```

Comment:
M: you can use backticks for naming tests https://stackoverflow.com/questions/45749349/kotlin-back-tick-escaping-in-method-names-how-does-it-work
Its much nicer to read

Reply:
Thanks. TIL.

Reply:
So I've tried the backticks naming, but if I understand it right it's supported from API level 30 and up. https://issuetracker.google.com/issues/208842981

And I'm testing with the min supported -> 21. And I don't know if there is a workaround for it besides bumping the min API version.

Reply:
Maybe this is related to Android UI test specifically then, not sure, maybe the Android folks can help here.
I'd avoid snake case tho, better `convertsUnknownValueToNull` if necessary, its more Java convention friendly.

Reply:
Converted to camel case.